lib/packages.cf

```diff
@@ -94,6 +94,28 @@ body package_module yum
     #default_options => {};
 }
 
+body package_module freebsd_ports
+# @brief Define details used when interfacing with the freebsd ports package
+# module.
+#
+# **Note:** Ports are expected to be setup prior to trying to use the packages
+# promise. You may need to ensure that `portsnap extract` has been run.
+#
+# **Example:**
+# ```cf3
+# bundle agent main
+# {
+#   packages:
+#     freebsd::
```

Comment:
There are other systems also called "ports" (OpenBSD and MacPorts to name two), so I think it's useful to distinguish that this is only for FreeBSD. I don't think this will work anywhere else – as far as I know there's no other OS that uses ports and pkgng (OpenBSD uses the "old-style" pkg_* not pkgng).